### PR TITLE
Fix for #11884: Move volume link to the source column and fix the link target

### PR DESCRIPTION
**Pull request by NomAnor**
This fixes the wrong link target in the volumes list of tasks. I moved the link to the source column which is the actual link target.

### CODE REVIEW

**Comment by tgross** [comment]
@ChaiWithJai this patch effectively changes the task index page to look like the following (the top row is a CSI volume and the bottom row is a host volume). I'm ok with this from a technical standpoint and it makes a lot more sense to me to have the link from the real source object, but I wanted to run it by you from a design standpoint before pressing the Big Green Merge Button 😀 

Screen Shot 2022-01-24 at 11 00 55 AM

**Review by tgross — APPROVED** [review]
LGTM

**Review by ChaiWithJai — APPROVED** [review]
LGTM

**Comment by tgross** [comment]
I've added a changelog entry and will merge once CI is green. Thanks @NomAnor!

**Pull request merged**
